src/services/tests/post.service.spec.ts

```diff
@@ -139,4 +139,334 @@ describe('PostService', () => {
       });
     });
   });
+
+  describe('savePost', () => {
+    it('should return post saved correctly if the relation does not exists', async() => {
+      const mockInput = {
+        userId: 'user-123',
+        save: true,
+        postId: 'post-1',
+      };
+
+      const mockFindUser = jest
+        .spyOn(prisma.user, 'findUnique')
+        .mockResolvedValue(mockTestUser);
+
+      const mockFindPost = jest
+        .spyOn(prisma.post, 'findUnique')
+        .mockResolvedValue(mockTestPost);
+
+      const mockAlreadyExistis = jest
+        .spyOn(prisma.userSavedPost, 'findUnique')
+        .mockResolvedValue(null);
+
+      const mockCreateUserSavedPost = jest
+        .spyOn(prisma.userSavedPost, 'create')
+        .mockResolvedValue({post_id: 'post-1',
+          createdAt: new Date(),
+          updatedAt: new Date(),
+          deletedAt: null,
+          user_id: 'user-123',
+        });
+
+      const result = await service.savePost(
+        mockInput.userId,
+        mockInput.postId,
+        mockInput.save,
+      );
+
+      expect(result).toEqual({
+        message: 'Post salvo com sucesso.',
```

Comment:
Service foi padronizado para ingles. Test esta dando erro por checar versão antiga.
```suggestion
        message: 'Post saved successfully.',
```